<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WatchBox Download</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icon</normaloff>:/icons/icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="entry_URL"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Insert">
         <property name="text">
          <string>Hinzuf√ºgen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MovieListWidget" name="list_URL"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Download">
       <property name="text">
        <string>Download Starten</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MovieListWidget</class>
   <extends>QListWidget</extends>
   <header>MovieList</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>entry_URL</tabstop>
  <tabstop>btn_Insert</tabstop>
  <tabstop>btn_Download</tabstop>
  <tabstop>list_URL</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
